source/common/access_log/access_log_formatter.cc

```diff
@@ -21,6 +21,27 @@ namespace AccessLog {
 
 static const std::string UnspecifiedValueString = "-";
 
+namespace {
+
+// Helper that handles the case when the ConectionInfo is missiring or if the desired value is
+// empty.
+StreamInfoFormatter::FieldExtractor sslConnectionStringInfoExtractor(
```

Comment:
Not sure if `sslConnectionInfoStringExtractor` sounds better.

Reply:
+1